tools/state-viewer/src/cli.rs

```diff
@@ -486,6 +491,19 @@ impl EpochInfoCmd {
     }
 }
 
+#[derive(clap::Args)]
+pub struct EpochAnalysisCmd {
+    /// The height of the epoch to analyze.
+    #[clap(long)]
+    height: EpochHeight,
```

Comment:
should this be the start height of the epoch or any height in that epoch?

Reply:
Start height of the epoch.